### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by jSchuetz88**
Update final Release Notes for Jupiter.

### CODE REVIEW

**Comment by jSchuetz88** [comment]
@stephanbcbauer @werner-roman @AnnKaCatX 
I've only updated the standards section of the releases notes. Are there any updates needed for the other sections?

**Comment by werner-roman** [comment]
Hey @jSchuetz88  - yes the Scope changes of Jupiter and the impact must also be also stated - @theresa is informed she is picking it up

**Comment by jSchuetz88** [comment]
@werner-roman @ther3sa 
I added a draft of the "scope clarification" to the release notes. Please review. Maybe more details / information are required regarding CX-0001 and CX-0006?

**Comment by ther3sa** [comment]
> @werner-roman @ther3sa I added a draft of the "scope clarification" to the release notes. Please review. Maybe more details / information are required regarding CX-0001 and CX-0006?

Should the release note rather go up to the top because of visibility or stay down?

File: src/pages/release-notes.md [hunk 1]
```diff
@@ -35,3 +35,11 @@ A comprehensive, coherent and compatible set of development artefacts. Highlight
 - several improvements to the user-friendliness of KITs based on feedback received to date, making Tractus-X overall more accessible and welcoming to newcomers and veterans alike.
 
 details to be found [here](https://eclipse-tractusx.github.io/CHANGELOG)
+
+### Further Scope Clarification
+
+To maintain the integrity and reliability of the onboarding process, the signing of self-descriptions (GXDC-compliance check) will be temporarily suspended:
```
Comment (werner-roman) [inline src/pages/release-notes.md]: GXDCH 

would write: "the signing of self-descriptions by the Gaia-X Digital Clearing House (GXDCH )will be temporarily suspended:"

File: src/pages/release-notes.md [hunk 2]
```diff
@@ -35,3 +35,11 @@ A comprehensive, coherent and compatible set of development artefacts. Highlight
...
+
+- This suspension only affects the CSP-B role within CX-0001 and CX-0006.
```
Comment (werner-roman) [inline src/pages/release-notes.md]: I would link the standards + details wich CAC are affected

**Review by werner-roman — CHANGES_REQUESTED** [review]

**Comment by werner-roman** [comment]
@jSchuetz88  "AIBSL" is wrong in the standards - it should be "AISBL"

**Review by ther3sa — APPROVED** [review]
Looks good to me

**Review by werner-roman — APPROVED** [review]
LGMT 🚀

**Comment by stephanbcbauer** [comment]
@jSchuetz88 Should we update the branch here?

**Pull request merged**